### PR TITLE
Release 2.7.0

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## JS-SDK 2.7.0
+
+### Push
+
+Support WebPush
```
Comment (wtrocki) [inline RELEASE_NOTES.md]: Can this be extended little bit.

Comment (danielpassos) [reply RELEASE_NOTES.md]: What kind of info should I add?

Comment (secondsun) [reply RELEASE_NOTES.md]: How about 
#### WebPush Support
 * WebPush Variants in mobile-services.json are now supported
 * Compatible browsers such as Chrome can register for and receive webpush push messages sent from UPS
 * Framework for connecting custom service-workers to notification APIs
 * Permissions for Notifications are handled by JS-SDK

Comment (danielpassos) [reply RELEASE_NOTES.md]: Done! @wtrocki Do you think we need some else info?

**Comment by wtrocki** [comment]
Works for me now!

**Pull request merged**